packages/hash/integration/src/tests/model/knowledge/orgMembership.model.test.ts

```diff
@@ -0,0 +1,96 @@
+import { getRequiredEnv } from "@hashintel/hash-backend-utils/environment";
+import { createGraphClient } from "@hashintel/hash-api/src/graph";
+import {
+  ensureWorkspaceTypesExist,
+  WORKSPACE_TYPES,
+} from "@hashintel/hash-api/src/graph/workspace-types";
+import {
+  OrgMembershipModel,
+  OrgModel,
+  OrgSize,
+  UserModel,
+} from "@hashintel/hash-api/src/model";
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import {
+  createKratosIdentity,
+  adminKratosSdk,
+} from "@hashintel/hash-api/src/auth/ory-kratos";
+
+jest.setTimeout(60000);
+
+const logger = new Logger({
+  mode: "dev",
+  level: "debug",
+  serviceName: "org-membership-tests",
+});
+
+const graphApiHost = getRequiredEnv("HASH_GRAPH_API_HOST");
+const graphApiPort = parseInt(getRequiredEnv("HASH_GRAPH_API_PORT"), 10);
+
+const graphApi = createGraphClient(logger, {
+  host: graphApiHost,
+  port: graphApiPort,
+});
+
+describe("OrgMembership model class", () => {
+  let kratosIdentityId: string;
+
+  let testUser: UserModel;
+
+  let testOrg: OrgModel;
+
+  beforeAll(async () => {
+    await ensureWorkspaceTypesExist({ graphApi, logger });
+
+    const testUserIdentity = await createKratosIdentity({
+      traits: {
+        emails: ["test@example.com"],
+      },
+    });
+
+    kratosIdentityId = testUserIdentity.id;
+
+    testUser = await UserModel.createUser(graphApi, {
+      emails: ["alice@example.com"],
+      kratosIdentityId,
+    });
```

Comment:
If you look in the other tests there's a function to do this. `createTestUser` in `hash/integration/src/tests/util.ts`, it's used in the other model classes

Reply:
Oh that's great, thanks! Completely missed this was added, I like it a lot. Made the change in https://github.com/hashintel/hash/pull/1068/commits/3823148f0cdfc50b69f2cd9ef08ba1f21fa49635, and have modified the method slightly to return the user model to avoid the caller from having to fetch it manually. Hope that makes sense.